Makefile

```diff
@@ -28,8 +28,6 @@ NPROCS ?= 1
 # to half the number of CPU cores.
 GO_TEST_PARALLEL := $(shell echo $$(( $(NPROCS) / 2 )))
 
-GO_INTEGRATION_TESTS_SUBDIRS = test
```

Comment:
The previous commit removes the v1alpha3 GKE cluster, which is the only thing these integration tests tested.